### PR TITLE
Fixes #14158 - Add tailoring file for scans

### PR DESCRIPTION
**Pull request by xprazak2**
Will need [this](https://github.com/OpenSCAP/ruby-openscap/pull/9) to work.

### CODE REVIEW

**Comment by xprazak2** [comment]
I added proxy plugin to foreman_infra, I think we need to add a webhook for jenkins in repo settings to enable testing on PRs.

**Comment by isimluk** [comment]
I just released ruby-openscap-0.4.7. You may want to amend your gemfile to depend on 0.4.7.

**Comment by xprazak2** [comment]
I updated the dependency version, thanks @isimluk!

File: lib/smart_proxy_openscap/openscap_plugin.rb [hunk 1]
```diff
@@ -21,6 +21,7 @@ class Plugin < ::Proxy::Plugin
                      :openscap_send_log_file => File.join(APP_ROOT, 'logs/openscap-send.log'),
                      :contentdir => File.join(APP_ROOT, 'openscap/content'),
                      :reportsdir => File.join(APP_ROOT, 'openscap/reports'),
-                     :failed_dir => File.join(APP_ROOT, 'openscap/failed')
+                     :failed_dir => File.join(APP_ROOT, 'openscap/failed'),
+                     :tailoringdir => File.join(APP_ROOT, 'openscap/tailoring')
```
Comment (ares) [inline lib/smart_proxy_openscap/openscap_plugin.rb]: Could you please rename this to `tailoring_dir`? I see there are others without underscore, but Ruby convention is to separate words.

**Comment by xprazak2** [comment]
I renamed `tailoringdir` to `tailoring_dir` and put back the old validation route with deprecation warning.

**Comment by ares** [comment]
I still see `tailoringdir` in the code. Have you opened the tracking redmine issue?

**Comment by xprazak2** [comment]
I opened the tracking issue and removed it from code - hopefully for good this time.

**Comment by ares** [comment]
:+1: works fine with older version of foreman_openscap and prints the deprecation warning, ready for merge

**Comment by ares** [comment]
[test]

**Comment by ares** [comment]
Thanks @domcleal, merging.

**Comment by ares** [comment]
[test] master is green again

**Comment by shlomizadok** [comment]
Thank you @xprazak2 !!
Thanks @ares for testing

**Pull request merged**